### PR TITLE
[DEV-11864] Add Input and Dropdown to Storybook

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

File: packages/ui/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuGroup,
+    DropdownMenuItem,
+    DropdownMenuLabel,
+    DropdownMenuSeparator,
+    DropdownMenuTrigger,
+} from '@/shadcn/components/ui/dropdown-menu';
```
Comment (oleg-semyonov) [inline packages/ui/src/components/Dropdown/Dropdown.tsx:13]: I don't quite get why there is an alias to `@/shadcn` that points to components that are using another library (radix-ui)? This makes things quite confusing IMO 🤔

**Review by oleg-semyonov — CHANGES_REQUESTED** [review]
I'm really confused with this PR. It introduces a lot of different things at once, and many parts of it just doesn't make sense and seem overly complex. Please see my comments

File: packages/ui/components.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
```
[the remark this answers is not in the document]
Comment (oleg-semyonov) [reply packages/ui/components.json:2]: On another note, I don't think that introduction of an external tool should be mixed in with a PR for a new UI component

File: packages/ui/src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (fgyimah) [reply packages/ui/src/components/Dropdown/Dropdown.tsx:13]: A part of the requirement for this library is for things like dropdown and other similar components, we don't implement from scratch but rather on top of other library and we decided to go with `shadcn`

File: packages/ui/package.json [hunk 3]
```diff
@@ -62,8 +62,13 @@
     "@heroicons/react": "^2.0.18",
     "@prezly/theme-kit-core": "^6.6.0",
     "@prezly/uploadcare-image": "^0.3.2",
+    "@radix-ui/react-dropdown-menu": "^2.0.6",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.0.0",
```
[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/package.json:67]: Will drop the others, they came with `shadcn` library install

File: packages/ui/components.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/components.json:2]: The way `shadcn` works is they don't provide a libray with components out of the box, but rather a cli that adds components built with `radix-ui` and tailwind, which can easily be customized

File: packages/ui/package.json [hunk 4]
```diff
@@ -62,8 +62,13 @@
...
     "date-fns": "^2.30.0",
+    "lucide-react": "^0.289.0",
```
[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/package.json:69]: This came with adding `shadcn` library, I would drop it.

File: packages/ui/src/@/lib/utils.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export function cn(...inputs: ClassValue[]) {
+    return twMerge(clsx(inputs));
+}
```
[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/src/@/lib/utils.ts:6]: It's supposed to support it, will update and drop `clsx` library

File: packages/ui/src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (oleg-semyonov) [reply packages/ui/src/components/Dropdown/Dropdown.tsx:13]: I don't see a value in this, honestly. We're already pulling a UI library (Radix UI), and I think we're competent enough to create our own components. If anything, we can simply copy-paste the initial code (which we'll customize anyway) if we need to.

File: packages/ui/components.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (oleg-semyonov) [reply packages/ui/components.json:2]: As I pointed in a comment above, I don't see any real value in this. Radix UI should be plenty enough to build our own components (which have their unique design anyway), and adding `shadcn` feels like unnecessary overhead.

[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/components.json:2]: Will drop `shadcn` and just build directly on top of Radix UI

File: packages/ui/src/components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (fgyimah) [reply packages/ui/src/components/Dropdown/Dropdown.tsx]: Dropped shadcn

File: packages/ui/src/components/Dropdown/Dropdown.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+];
+
+const DropdownTemplate: StoryFn<typeof Dropdown> = (args) => (
+    <Dropdown {...args}>
+        <Dropdown.Group>
+            {ITEMS.map((item) => (
+                <Dropdown.Item
+                    className="label-medium px-4 py-0 mb-4 hover:bg-transparent focus:bg-transparent cursor-pointer"
+                    key={item}
+                >
```
[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/src/components/Dropdown/Dropdown.stories.tsx]: Updated

File: packages/ui/src/base.styles.css [hunk 7]
```diff
@@ -5,6 +5,79 @@
...
+        --border: 215 27.9% 16.9%;
+        --input: 215 27.9% 16.9%;
+        --ring: 216 12.2% 83.9%;
+    }
+}
+
+@layer base {
+    * {
+        @apply border-border;
+    }
```
[the remark this answers is not in the document]
Comment (fgyimah) [reply packages/ui/src/base.styles.css]: Fixed

**Review by oleg-semyonov — APPROVED** [review]

**Pull request merged**